$OpenBSD$

Use unsigned char instead of int for evas_object_color_get()

--- src/bin/e_slidecore.c.orig	Wed Nov  2 12:22:09 2011
+++ src/bin/e_slidecore.c	Wed Nov  2 12:22:21 2011
@@ -187,7 +187,7 @@ _e_smart_reconfigure(E_Smart_Data *sd)
 {
    Evas_Coord dp, pos;
    int p1, p2, at, pl1, pl2, n;
-   int r, g, b, a;
+   unsigned char r, g, b, a;
 
    evas_object_move(sd->event_obj, sd->x, sd->y);
    evas_object_resize(sd->event_obj, sd->w, sd->h);
